applications/Interstellar-Network.md

```diff
@@ -0,0 +1,575 @@
+- **Confirm a transaction with ONLY ONE SCREEN** No short text message (SMS) to wait for, no additional 2FA app (like Google Authenticator) to use, no QR code to scan.
+Anyway, banking trojan can already compromise the previous 2FA schemes*
+[Android banking Trojan to steal cryptocurrency and 2FA codes](https://www.revelock.com/en/blog/oscorp-android-banking-trojan-to-steal-cryptocurrencies-and-2fa-codes  )
+
+
+ **You just need to input on the keypad the one-time code you see (here 256)**
+
+**That's all to securely confirm a transaction** ![Wallet-superposition-White-Shadow4medium smaller](https://user-images.githubusercontent.com/4605611/141301152-951143c3-d820-4bc1-8120-5a60a6878d52.png)
+> And the best part, this scheme is designed to resist malware attacks from state of the art banking trojans (and even targeted attacks at a later stage)
+
+
+- **Decentralized key & asset management service** where user’s privates key and signature program are stored and executed in Hardware Enclave/Trusted Execution Environment TEE*
+
+- **Social Recovery Service** that leverages the existing substrate pallet, and a novel decentralized autonomous recovery service* (Comment: we hope that we will be able to provide a response to the related RFP in the following phases)
+- **Features to securely send coins with social network messages (even to people with no-wallet)** Link to "Can be easy to set-up wallet an efficient customer acquisition tool for DeFi players" to add
+- **Cross chain Swap feature** based on a Continious Liquitity Pool (CLP like ThorChain but with higher security and performance TEE/MPC/TTS + Trusted Transaction Validation protocol with mutisig option*  mobiles + yubikey for Validators and power users)
+
+
+#### Substrate Technology Overview
+
+**The solution relies on a substrate blockchain and on substaTEE/IntegriTEE Workers** to secure private keys, signature programs and the friendly authentication and transaction validation service. The last service use a novel **Decentralized Trusted Transaction Validation Protocol** that leverage **TEE features on mobile**, combined with **One Time Garbled Circuit and Visual Cryptography** to provide a **Trusted Authentication and Trusted UI layer** on user devices.
+
+***Link to medium Interstellar Wallet – Technology and security lightpaper.(draft/work in progress)**
+
+#### Interstellar project Phase 1:
+
+The project phase 1 focus on two of the core components of the Interstaller solution: An OCW garbled circuit factory GCF (to manage an external Garbled Circuit Genrator service) and an implementation of the transaction validation protocol in substrate pallets. It demonstrate the usage of GCF whithin a substrate framework and with a mobile  Garbled Circuit evaluator client. We designed this Garbled circuit Factory to be used by substrate devellopers regardless of Interstellar solution.
```

Comment:
typos

Reply:
"We designed this Garbled circuit Factory to be used by substrate devellopers regardless of Interstellar solution." attention, il vaut peut etre mieux eviter de trop s'engager?